presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -603,6 +659,27 @@ private void testUnsupportedDataType(String databaseDataType)
         }
     }
 
+    public static DataType<ZonedDateTime> prestoTimestampWithTimeZoneDataType()
+    {
+        return dataType(
+                "timestamp with time zone",
+                TIMESTAMP_WITH_TIME_ZONE,
+                DateTimeFormatter.ofPattern("'TIMESTAMP '''yyyy-MM-dd HH:mm:ss.SSS VV''")::format,
```

Comment:
These patterns could be constants

Reply:
Could be, but I'd prefer to define them inline. It's easier to reason about.